runtime/v2/runner/src/test/resources/com/walmartlabs/concord/runtime/v2/runner/errorBlockScoping/concord.yml

```diff
@@ -0,0 +1,23 @@
+configuration:
+  runtime: "concord-v2"
+
+flows:
+  default:
+    - call: anotherFlow
+      out:
+        - myVar
+    - log: "Hello, ${myVar}!"
+
+  anotherFlow:
+    - try:
+        - call: createMyVar
+          out:
+            - myVar
+        - throw: kapow! kamblamo!
+      error:
+        - log: "caught an error"
+    - log: "myVar should exist: ${myVar}"
```

Comment:
This should still work even when the `throw` call is omitted

Reply:
Right, I initially though the issue was limited to the `error` blocks. I'll update the PR tomorrow or on Wednesday.

Reply:
Another similar scenario. Putting an `error` block on a flow call messes up the variable scope too.
```yaml
flows:
  default:
    - call: anotherFlow
      out:
        - myVar
    - log: "Hello, ${myVar}!"

  anotherFlow:
    - call: createMyVar
      out:
        - myVar
      error:
        - log: "caught an error"
    - log: "myVar should exist: ${myVar}"  # doesn't work

  createMyVar:
    - set:
        myVar: 'world'
```